core/domain/services/admin/events/editor/EventManagers.php

```diff
@@ -116,6 +135,12 @@ private function getEventManagerUsers(array $event_manager_roles)
                 $operator = 'OR';
             }
         }
+        foreach ($capabilities as $capability) {
+            // for each capability, add a WHERE clause
+            $SQL     .= $operator . ' u2.meta_value LIKE \'%"' . $capability . '";b:1;%\' ';
```

Comment:
`$capability` does not need to be SQL escaped, right?

Reply:
no the capabilities are already coming from the db and can't have quotes or special characters in their names (afaik 🤔 )